components/table/stories/template.js

```diff
@@ -80,19 +113,23 @@ export const TableRowItem = ({
 		aria-selected=${ifDefined(showCheckbox ? "true" : undefined)}
 		data-tier=${ifDefined(tier)}
 		?hidden=${isHidden}
+		tabindex="0"
 	>
 		${when(showCheckbox && !isSectionHeader, () => html`
 			<${cellTag}
 				role="gridcell"
 				class=${classMap({
 					[`${rootClass}-cell`]: true,
 					[`${rootClass}-checkboxCell`]: true,
+					[`${rootClass}-cell--alignEnd`]: getTextAlignment(0) === "end",
+					...cellCustomClasses?.[0]?.reduce((a, c) => ({ ...a, [c]: true }), {}),
```

Comment:
I wanted to show the individual cell focus states, so I had to pass custom classes to the cells.